_internals/configuration.md

```diff
@@ -193,391 +39,321 @@ Sets the first day of the week to a number from 1 through 7.
 
-### `babelfishpg_tsql.trace_exec_codes`
+### `babelfishpg_tds.tds_default_packet_size`
 
-Trace execution code of iterative executor.
+This parameter specifies the default packet size for all the clients being connected. Accepted values range from 512 to 32767; the default value is 4096.
 
 
-### `babelfishpg_tsql.trace_exec_counts`
+### `babelfishpg_tds.tds_default_protocol_version`
 
-Trace execution count of each code for iterative executor.
+This parameter specifies the default TDS protocol version for all the clients being connected. The accepted values are:
 
+- TDSv7.0
+- TDSv7.1
+- TDSv7.1.1
+- TDSv7.2
+- TDSv7.3A
+- TDSv7.3B 
+- TDSv7.4 
+- DEFAULT
 
-### `babelfishpg_tsql.trace_exec_time`
 
-Trace execution time of each code for iterative executor.
+### `babelfishpg_tds.tds_ssl_encrypt`
 
+This parameter specifies the SSL encryption used by Babelfish. The default value is `false`.
 
-### `babelfishpg_tsql.textsize`
 
-Set `TEXTSIZE`.
+### `babelfishpg_tds.tds_ssl_max_protocol_version`
 
+Sets the maximum SSL/TLS protocol version to use for the TDS session. The default value is 'TLSv1.2'; accepted values are:
 
-### `babelfishpg_tsql.use_antlr`
+- TLSv1
+- TLSv1.1
+- TLSv1.2
 
-This parameter is now obsolete, and will be removed in a future release.
-There is no way to switch parsers, both yacc and antlr parsers are required.
 
-For more information see [Issue #47](https://github.com/babelfish-for-postgresql/babelfish_extensions/issues/47#issuecomment-994107040).
+### `babelfishpg_tds.tds_ssl_min_protocol_version`
 
-The default value is `true`.
+Sets the maximum SSL/TLS protocol version to use for the TDS session. The default value is 'TLSv1'; accepted values are:
 
+- TLSv1
+- TLSv1.1
+- TLSv1.2
 
-### `babelfishpg_tsql.version`
 
-Sets the output of the `@@VERSION` variable.
+### `babelfishpg_tds.trigger_fault_enabled`
 
+This parameter controls fault injection triggers.
 
-### `babelfishpg_tsql.xact_abort`
 
-Control transactional behavior when an error happens.
+### `babelfishpg_tds.unix_socket_directories`
 
+The UNIX socket directory. The default is /tmp.
 
-### `babelfishpg_tds.default_server_name`
 
-Predefined Babelfish default server name.
+### `babelfishpg_tds.unix_socket_group`
 
+This parameter controls the UNIX socket group setting.
 
-### `babelfishpg_tds.listen_addresses`
 
-Sets the host name or IP address(es) for TDS to listen on.
+### `babelfishpg_tds.unix_socket_permissions`
 
+This parameter controls UNIX socket permissions. The default is 0700.
 
-### `babelfishpg_tds.port`
 
-The port to use for the TDS protocol (not standard PostgreSQL).
 
-### `babelfishpg_tds.set_db_session_property`
+## Parameters that should not be modified
 
-Set database session property on TDS connections. What it means is that
-`babelfishpg_tsql.database_name` is set during the login process. An error will
-happen if the `database_name` is not set.
+You may come across these parameters in the source code, but you shouldn't change them:
```

Comment:
Shouldn't we just list these out and not document them? Documenting them adds additional maintenance burden and invites people to mess with these parameters.

Reply:
Done - I've removed the comments and reformatted this section as a bulleted list...